ovn_connect_test.go

```diff
@@ -45,32 +46,50 @@ const (
 	UNIX                 = "unix"
 )
 
-var ovndbapi Client
+var (
+	ovndbapi   Client
+	ovn_db     string
+	ovn_socket string
+)
+
+// Default connect to ovn nb db
+func getOVNDB() (db string) {
```

Comment:
Sorry I don't understand how this works. Usually we will set both OVN_NB_DB and OVN_SB_DB envs to point to corresponding URLs. If both are set, this always returns dbSB? This seems strange.